client/dashboard/sites/overview/visibility-card.tsx

```diff
@@ -0,0 +1,113 @@
+import { CircularProgressBar } from '@automattic/components';
+import { __experimentalText as Text } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { lockOutline, published } from '@wordpress/icons';
+import { launch } from '../../components/icons';
+import { isSelfHostedJetpackConnected } from '../../utils/site-types';
+import OverviewCard from '../overview-card';
+import type { Site } from '../../data/types';
+
+const CARD_PROPS = {
+	title: __( 'Visibility' ),
+	trackId: 'visibility',
+};
+
+function getVisibilityURL( site: Site ) {
+	if ( isSelfHostedJetpackConnected( site ) ) {
+		return undefined;
+	}
+
+	return `/sites/${ site.slug }/settings/site-visibility`;
+}
+
+function VisibilityCardUnlaunched() {
+	const isSetupComplete = true;
+	let heading = __( 'Coming soon' );
+	let description = __( 'Finish setting up your site' );
+
+	if ( isSetupComplete ) {
+		heading = __( 'Launch site' );
+		description = __( 'Ready to go public?' );
+	}
+
+	return (
+		<OverviewCard
+			{ ...CARD_PROPS }
+			icon={ launch }
+			heading={ heading }
+			description={ description }
+			sideContent={
+				<CircularProgressBar
+					currentStep={ 5 }
+					numberOfSteps={ 5 }
+					size={ 80 }
+					strokeColor="var(--wp-admin-theme-color)"
+					strokeWidth={ 1.5 }
+					variant="success"
+					customText={
+						<Text lineHeight="20px" size={ 15 } weight={ 500 }>
+							5/5
+						</Text>
+					}
+				/>
+			}
+		/>
+	);
+}
+
+function VisibilityCardComingSoon( { site }: { site: Site } ) {
+	return (
+		<OverviewCard
+			{ ...CARD_PROPS }
+			icon={ launch }
+			heading={ __( 'Coming soon' ) }
+			description={ __( 'Ready to go public?' ) }
```

Comment:
It feels weird to have this copy for staging sites. Are staging sites meant to be public 🤔 cc: @matt-west

Reply:
True, I think when every piece is in place we'll have a better idea on what makes sense or not given context. Also, I'm assuming that usability tests will paint a more complete picture of what users are expecting to see.